dht/i_partitioner.cc

```diff
@@ -204,7 +204,7 @@ ring_position_range_sharder::next(const schema& s) {
     return ring_position_range_and_shard{std::move(_range), shard};
```

Comment:
this could also happen when refreshing data in a live system. vnode doesn't have file streaming, tablet does but it isn't susceptible since with it, there's a single range per sstable.